notes/returned-mail-received.vbs

```diff
@@ -55,26 +56,36 @@ changelog_display
 'END CHANGELOG BLOCK =======================================================================================================
 
 'THE SCRIPT-----------------------------------------------------------------------------------------------------------------
-'Connects to BLUEZONE
-EMConnect ""
-'Grabs the MAXIS case number
-CALL MAXIS_case_number_finder(MAXIS_case_number)
+
+EMConnect ""                                        'Connecting to BlueZone
+CALL MAXIS_case_number_finder(MAXIS_case_number)    'Grabbing the CASE Number
+call Check_for_MAXIS(false)                         'Ensuring we are not passworded out
+back_to_self                                        'added to ensure we have the time to update and send the case in the background
 
 '-------------------------------------------------------------------------------------------------DIALOG
-Dialog1 = "" 'Blanking out previous dialog detail
-'Running the initial dialog
-BeginDialog Dialog1, 0, 0, 211, 85, "Returned Mail"
-  EditBox 55, 5, 40, 15, MAXIS_case_number
-  EditBox 155, 5, 50, 15, date_received
-  DropListBox 90, 25, 115, 15, "Select One:"+chr(9)+"forwarding address in MN"+chr(9)+"forwarding address outside MN"+chr(9)+"no forwarding address provided"+chr(9)+"no response received", ADDR_actions
-  EditBox 90, 45, 115, 15, worker_signature
+Dialog1 = "" 'Running the initial dialog
+BeginDialog Dialog1, 0, 0, 221, 215, "RETURNED MAIL PROCESSING"
+  EditBox 60, 5, 50, 15, MAXIS_case_number
+  EditBox 60, 25, 50, 15, date_received
+  DropListBox 105, 65, 110, 15, "Select:"+chr(9)+"forwarding address in MN"+chr(9)+"forwarding address outside MN"+chr(9)+"no forwarding address provided"+chr(9)+"no response received", ADDR_actions
+  DropListBox 100, 150, 110, 15, "Select:"+chr(9)+"YES"+chr(9)+"NO-attempted"+chr(9)+"NO-not attempted", client_contact_dropdown 'TO DO question do we want more HERE'
+  EditBox 70, 175, 145, 15, worker_signature
   ButtonGroup ButtonPressed
-    OkButton 110, 65, 45, 15
-    CancelButton 160, 65, 45, 15
-  Text 5, 30, 85, 10, "Mail Has Been Returned:"
-  Text 5, 50, 60, 10, "Worker Signature:"
+    OkButton 115, 195, 50, 15
+    CancelButton 165, 195, 50, 15
+    PushButton 135, 5, 80, 15, "SNAP TE02.08.012", SNAP_POLI_TEMP_button
+    PushButton 135, 25, 80, 15, "CASH TE02.08.011", CASH_POLI_TEMP_button
+    PushButton 135, 45, 80, 15, "ONEsource", one_source_button
   Text 5, 10, 50, 10, "Case Number:"
-  Text 100, 10, 50, 10, "Date Received:"
+  Text 5, 30, 50, 10, "Date Received:"
+  Text 5, 70, 85, 10, "Mail Has Been Returned:"
+  GroupBox 5, 85, 210, 85, " Request for Contact"
```

Comment:
I am not sure about the verbiage here. What are we trying to accomplish with having this information listed here. Encouraging contact with the resident could be beneficial but I don't know that this is supportive.

Reply:
Request for contact is outlined specifically for HC programs per conversation with Ilse I will update the script for the contact requirement to be program specific

Reply:
discussed with Casey and updated verbiage